src/openforms/registrations/contrib/zgw_apis/models.py

```diff
@@ -57,6 +57,14 @@ class ZgwConfig(SingletonModel):
         validators=[validate_rsin],
         help_text=_("Default RSIN of organization, which creates the ZAAK"),
     )
+    medewerker_roltype = models.URLField(
+        _("medewerker roltype"),
+        blank=True,
+        help_text=_(
+            "URL of the ROLTYPE to use for employees filling in a form for a citizen/company."
+        ),
+        max_length=1000,
+    )
```

Comment:
I honestly don't see a point in making these global defaults (the same applies to existing fields like `zaaktype` etc.), so let's only handle them in the configuration options serializer for the plugin.